src/main/java/edu/ucla/library/prl/harvester/services/HarvestServiceUtils.java

```diff
@@ -83,6 +84,18 @@ final class HarvestServiceUtils {
      */
     private static final Pattern ITEM_URL_FIELD_PATTERN = Pattern.compile("identifier(?:\\..+)?");
 
+    /**
+     * The pattern for URL paths that have the filetype extension for an image.
+     */
+    private static final String IMAGE_FILETYPE_EXTENSION = ".+\\.(?:(?:avif)|(?:gif)|(?:jpe?g)|(?:png)|(?:webp))$";
+
+    /**
+     * A Collector that partitions a list of URLs by whether or not they have the filetype extension for an image.
+     */
+    @SuppressWarnings("PMD.UseLocaleWithCaseConversions")
```

Comment:
Justification: when matching against a pattern that only ever contains ASCII chars, the implicit default locale for lowercasing will always suffice.

Reply:
I imagine that using a specific US locale would use less characters than the SuppressWarnings exception? So, while I see what you say is true, I'm not sure of the value of using the exception rather than using the best practice(?) Not a big deal though.

Reply:
Changed to follow best practice since it's so easy to do 🙃